doc/overview/dynamicCapabilities.md

```diff
@@ -0,0 +1,122 @@
+Dynamic Capabilities
+====================
+
+Project capabilities are the recommended way to determine the type, platform,
+and features of a project. In the modern world, features of a project can change over the time.
+For example, when a NuGet package is added.  In the next version of the Visual Studio,
+the CPS has a built-in infrastructure to support capabilities of a project to be changed without
+reloading the project.
+
+This infrastruture will eventually allow design time experience of
+a project adjusted dynamically based on features/NuGet packages being used in the project.
+For example, WPF features can be turned on, only the project is referencing WPF related packages,
+instead of basing on which template is used to create the project.
```

Comment:
Perhaps past tense? "instead of depending on which template was use when the project was created"